source/_integrations/legrand_homepluscontrol.markdown

```diff
@@ -0,0 +1,103 @@
+
+This call retrieves the information of the plants that are associated to a user. It is assumed that this information does not change very often. For the most part, a user will have a single *Legrand Home+ Control* gateway in their home, i.e. a single *plant*, and this will not change frequently.
+
+For this reason, the default polling interval for the plant information is set to a large value: 7200 seconds (2 hours).
+
+### Plant Topology Request
+
+This call retrieves the *layout* of the devices that make up a *plant*. This provide the list of device, their names and some additional information of the devices (but not their status).
+
+It is assumed that plant topology is not changing very often, so the default polling interval for this information is set to a default value of 3600 seconds (1 hour).
+
+### Module Status Request
+
+This call retrieves the current operating status of the devices that make up the plant's topology - this includes the *reachability* of the device and its on/off status. 
+
+This information is expected to change frequently, so the polling interval is set to a lower default value of 300 seconds (5 minutes). This is still not as frequent as would be ideal, but is still reasonable considering the API calls-per-day limitation.
+
+
+## API Nomenclature
+
+Within the context of the `Home+ Control` API you may come across the following terms:
+* *Plant*: This is the term used to represent a *home* that holds the Legrand devices. In practice, a *plant* is represented by the *Legrand Home+ Control* gateway that acts as the central hub of the rest of the devices in the home network (uses Zigbee).
+* *Module*: This is the term used to represent a generic device within the *plant*, i.e. a light, a plug, a remote, etc.
+* *Light*: This is the term used to represent a light switch (or a micro-module). It is not modeled as your usual light entity because there are no brightness, color, etc. controls. It is modeled as a simple on/off switch component.
+* *Plug*: This is the term used to represent a power outlet.
+
+Other devices that are mentioned in the API, but that are not currently supported by this integration are: *remotes* (wireless switches), *heaters* and *automations*.
+
+## API Limitations
+
+As mentioned in the [authentication](#authentication) section, this integration requires you to set up a subscription in the *Works with Legrand* platform. 
+
+Currently, end-users only have access to the *Starter Kit* subscription which has a major limitation in the number of allowed API requests that are allowed - only 500 API calls per day (counter is reset at 00:00 every day).
+
+This means that the integration has to set default polling intervals that are considerably larger than one would want for a Home Assistant integration. These intervals are [configurable](#polling-interval-configuration), but take care not to exceed the daily quota to prevent the integration from failing with `403 Forbidden` HTTP responses.
```

Comment:
>  These intervals are [configurable](#polling-interval-configuration)

However, it is not explained how one can do that?

Reply:
I have added more information in the section that describes how these values are configured. Hopefully this is clearer now in the text.